packages-ts/gauntlet-terra-contracts/src/commands/abstract/polymorphic.ts

```diff
@@ -0,0 +1,64 @@
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { MultisigTerraCommand } from '../contracts/multisig'
+import { Result, Command } from '@chainlink/gauntlet-core'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+import DeployLink from '../../commands/contracts/link/deploy'
+import AbstractCommand from '.'
+import { makeAbstractCommand } from '.'
+
+class EmptyCommand extends Command {
+  constructor() {
+    super({ help: false }, [])
+  }
+}
+
+type ICommandConstructor = (flags: any, args: string[]) => void
```

Comment:
Why this "polymorphic" command? Why do we need it? We are already wrapping the commands into a multisig command